test/infrastructure/inmemory/templates/clusterclass-in-memory.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: ClusterClass
+metadata:
+  name: in-memory
```

Comment:
proposing some alternative names: quick-start, development, or stress-test

Reply:
Probably good if this ClusterClass and its objects don't clash with the one from CAPD. Maybe quick-start-in-memory?

(they aber both "just" dev providers, but would be not too bad if we can deploy them at the same time in the same mgmt cluster)

Reply:
Will go with `stress-test`.

Reply:
I think quick-start-in-memory would have been better. Especially as it will show up in the list of release attachments:

* cluster-template-development.yaml (CAPD)
* clusterclass-quick-start.yaml (CAPD)
* cluster-template-inmemory-development.yaml (CAPIM)

And then:
* clusterclass-stress-test.yaml or
* clusterclass-quick-start-in-memory.yaml (or maybe clusterclass-in-memory-quick-start-.yaml)

I think it would be nice if ithe YAMLs clearly match to a specific provider

Reply:
@fabriziopandini @ykakarap WDYT? If you prefer "stress-test" I won't block.

Reply:
my comment was about the name of the object in K8s, and I would suggest to use `in-memori-quick-start`

but also @sbueringer point about the name of the file in the release artifact folder makes sense, so let's call the file `clusterclass-in-memory-quick-start.yaml`

Reply:
Will rename it "in-memory-quick-start". I like that the file names will be consistent.

Reply:
>my comment was about the name of the object in K8s, and If I have to pick one I would suggest to use in-memory-quick-start

The name of the file has to match the name of the clusterclass. clusterctl needs the files to follow a [naming convention](https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#naming-conventions-2) for `clusterctl generate cluster` to work.